src/ios/BluetoothLePlugin.m

```diff
@@ -475,46 +476,46 @@ - (void)notify:(CDVInvokedUrlCommand *)command {
 - (void)peripheralManagerDidUpdateState:(CBPeripheralManager *)peripheral {
   NSString* error = nil;
   switch ([peripheral state]) {
-    case CBPeripheralManagerStatePoweredOff: {
+    case CBManagerStatePoweredOff: {
       error = logPoweredOff;
       break;
     }
-
-    case CBPeripheralManagerStateUnauthorized: {
+      
```

Comment:
Mind removing the extra whitespace?

Reply:
I can try, this was IDE doing its thing with automatic formatting :(

Reply:
Should be done now.